File: Level1/이상한 문자 만들기.cpp
#include <string>
#include <vector>

using namespace std;

string solution(string s) {
    string answer = "";
    int index = 0;
    
    for(int i = 0; i < s.size(); i++)
    {
		if (s[i] != ' ')
		{
			if (index % 2 == 0)
				answer.push_back(toupper(s[i]));
			else
				answer.push_back(tolower(s[i]));
			index++;
		}
		else
		{
			index = 0;
			answer.push_back(' ');
		}
    }
    return answer;
}

File: Level1/이상한 문자 만들기.js
function solution(s) {
    let answer = '';
    let count = 0;
    
    for(let i = 0; i < s.length; i++){
        if(s[i] === " "){
            answer += " ";
            count = 0;
            continue;
        }
        
        if(count % 2){
            //odd
            answer += s[i].toLowerCase();
        }
        else{
            //even
            answer += s[i].toUpperCase();
        }
        count ++;
    }
    return answer;
}